setup.cfg

```diff
@@ -47,6 +46,8 @@ examples =
     boost-histogram
```

Comment:
You added the [examples] target. Maybe you'd like to mention this in the readme, where you suggest to look at the example?